### ISSUE TITLE
trouble with adding modded fish from another mod (fishofthieves) to work with this one

### ISSUE THREAD

**Issue by Sordrin** [Pyxrs/Live-Fishing#11]
I tried following the json format to allow fish from this mod fish of thieves [https://github.com/SteveKunG/FishOfThieves](url) to work with your mod as I saw it allows for the addition of modded fish, except it continues to only spawn vanilla fish despite me adding a json file with the correct format and entity/item ids in the data/fishing/fishing folder using winrar and notepad++ (saved as a json file). I also tried using a script editor mod (Kubejs) and put the same code in a similar file path except in the Kubejs folder and it still wouldn't run and I'm not quite sure what I'm doing wrong. Here's what the code looks like which as you can see I just replaced the original entity and item tags with that of some of the modded fish tags. The fish do have different variants but I figured that's not the problem as the mod works fine with tropical fish despite tropical fish having several variants as well.

`{
  "replace": false,
  "fishing": [
    {
      "type": "fishing:basic",
      "item": {
        "item": "fishofthieves:splashtail",
        "count": 1
      },
      "entity": "fishofthieves:splashtail"
    },
    {
      "type": "fishing:basic",
      "item": {
        "item": "fishofthieves:pondie",
        "count": 1
      },
      "entity": "fishofthieves:pondie"
    },
    {
      "type": "fishing:basic",
      "item": {
        "item": "fishofthieves:islehopper",
        "count": 1
      },
      "entity": "fishofthieves:islehopper"
    },
    {
      "type": "fishing:basic",
      "item": {
        "item": "fishofthieves:ancientscale",
        "count": 1
      },
      "entity": "fishofthieves:ancientscale"
    }
  ]
}`

**Comment by Pyxrs** [comment]
Yes, this used to be a feature that was added by a contributor, Jab125. However, I am not able to maintain the feature because I don't understand the code, and it had caused a lot of problems. I decided to remove data driven stuff in the latest release, but forgot to remove it from the documentation. Sorry!

Leaving this open if anybody wants to add it back
